src/PolyhedralGeometry/PolyhedralComplex/constructors.jl

```diff
@@ -0,0 +1,86 @@
+###############################################################################
+###############################################################################
+### Definition and constructors
+###############################################################################
+###############################################################################
+
+struct PolyhedralComplex
+    pm_complex::Polymake.BigObject
+    function PolyhedralComplex(pm::Polymake.BigObject)
+        return new(pm)
+    end
+end
+
+pm_object(pc::PolyhedralComplex) = pc.pm_complex
+
+
+@doc Markdown.doc"""
+    PolyhedralComplex(Polytopes, VR, far_vertices, Lineality)
+
+# Arguments
+- `Polytopes::IncidenceMatrix`: An incidence matrix; there is a 1 at position
+  (i,j) if the ith polytope contains point j and 0 otherwise.
+- `VR::Matrix`: The points whose convex hulls make up the polyhedral
+  complex. This matrix also contains the far vertices.
+- `far_vertices::Vector{Int}`: Vector containing the indices of the rows
+  corresponding to the far vertices in `VR`.
+
+A polyhedral complex formed from points, rays, and lineality combined into
+polyhedra indicated by an incidence matrix, where the columns represent the
+points and the rows represent the polyhedra.
+
+# Examples
+```jldoctest
+julia> IM = IncidenceMatrix([[1,2,3],[1,3,4]])
+2×4 IncidenceMatrix
+[1, 2, 3]
+[1, 3, 4]
+
+
+julia> VR = [0 0; 1 0; 1 1; 0 1]
+4×2 Matrix{Int64}:
+ 0  0
+ 1  0
+ 1  1
+ 0  1
+
+julia> PC = PolyhedralComplex(IM, VR)
+A polyhedral complex in ambient dimension 2
+```
+"""
+function PolyhedralComplex(Incidence::IncidenceMatrix, VR::Union{SubObjectIterator{<:Union{PointVector,PointVector}}, Oscar.MatElem, AbstractMatrix}, far_vertices::Union{Vector{Int}, Nothing} = nothing, L::Union{SubObjectIterator{<:RayVector}, Oscar.MatElem, AbstractMatrix, Nothing} = nothing)
```

Comment:
Please wrap this across multiple lines. While we don't impose a fixed line width limit, as a rule of thumb, anything that forces me to scroll horizontally when reviewing a PR in the GitHub web interface is bad. If you stick to 80-120 characters per line, that's fine, anything beyond probably isn't.